src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -137,6 +165,18 @@ private void LogConstant(Optional<object> constant, string header = "Constant")
         private void LogConstant(object constant, string header = "Constant")
         {
             var valueStr = constant != null ? constant.ToString() : "null";
+            if (constant is string)
+            {
+                if (valueStr == string.Empty)
+                {
+                    valueStr = @"""""";
+                }
+                else
+                {
+                    valueStr = @"""" + valueStr + @"""";
+                }
```

Comment:
Is the `if` case necessary? The `else` should handle the `string.Empty` case as well.